src/main/java/pl/psnc/dei/iiif/Converter.java

```diff
@@ -53,6 +59,18 @@
 	@Autowired
 	private RecordsRepository recordsRepository;
 
+	@Autowired
+	private ContextMediator contextMediator;
+
+	@Autowired
+	private PersistableExceptionService persistableExceptionService;
+
+	@Autowired
+	private ConversionDataHolderTransformer conversionDataHolderTransformer;
```

Comment:
are injected beans `persistableExceptionService` and `conversionDataHolderTransformer` used anywere?

Reply:
removed